src/app/components/game-center/game-center.component.ts

```diff
@@ -108,6 +110,10 @@ export class GameCenterComponent implements OnInit {
       this.writeInSelectedBox(key);
       this.rewriteWord();
       this.correctSyntaxWord = this.isSyntaxCorrect();
+      this.selectResultBox = this.selectResultBox + 1;
+      if (this.selectResultBox > 5) {
+        this.selectResultBox = 5;
+      }
```

Comment:
```suggestion
     this.selectResultBox = this.selectResultBox === MaxResultBox ? MaxResultBox: ++this.selectResultBox
```
Using the const vars declared earlier